RocketChat/Rocket.Chat
# [FIX] Attachments not rendered if deployed on subfolder

## Proposed changes (including videos or screenshots)





## Issue(s)

#18759
#19542

## Steps to test or reproduce

- Run develpment server with this command `ROOT_URL=http://localhost:3000/<SUBFOLDER_NAME> meteor`
- Go to Setting > General and change the Site URL setting to match the new address.
- Avatars and attachments should be working
PS.: Old quotes will have the avatars broken, since the data from quotes is simply copied instead of referenced. This will only affect servers that CHANGED URL to run on a subfolder
## Further comments
